fdbserver/RestoreUtil.h

```diff
@@ -34,6 +34,12 @@
 #include <cstdint>
 #include <cstdarg>
 
+#define FR_DEBUG_SEVERITY 		  SevDebug
```

Comment:
I think that we probably don't need to align these, but if you do, it should be done with spaces rather than tabs so the alignment doesn't depend on tab width.

Reply:
Is there any expectation that these severities will be used outside of the trace event macros below? If not, then it's probably not necessary to define them separately.